frontend/packages/ceph-storage-plugin/src/features.ts

```diff
@@ -52,20 +53,25 @@ export enum GUARDED_FEATURES {
   OCS_NAMESPACE_STORE = 'OCS_NAMESPACE_STORE',
   ODF_MCG_STANDALONE = 'ODF_MCG_STANDALONE',
   ODF_HPCS_KMS = 'ODF_HPCS_KMS',
+  SS_LIST = 'SS_LIST',
```

Comment:
Isn't add-capacity modal also migrated in the same PR ?

Reply:
ack, missed out.